code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -496,6 +511,12 @@
 			return 0
 	return 1
 
+/mob/living/simple_animal/proc/attack_threshold_check(damage, damagetype = BRUTE)
+	if(damage <= force_threshold || ignored_damage_types[damagetype])
+		visible_message("<span class='warning'>[src] looks unharmed.</span>")
+	else
+		adjustBruteLoss(damage)

```

Comment:
You could probably just use a switch statement for damagetype. Can't 100% recall the BYOND syntax for it, but it should look something like:
```
switch(damagetype)
     if(BRUTE)
          adjustBruteLoss(damage)
     if(BURN)
          adjustBurnLoss(damage)
```
etc.